### PR TITLE
Post a new comment to an existing article

### PR DESCRIPTION
**Pull request by golovko**

### CODE REVIEW

File: src/components/PostComment.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const saveComment = (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+    postComment(article_id, {
+      username: username,
+      body: comment,
+    })
+      .then((response) => {
+        if (response.status === 201) {
+          alert('Posted');
```
Comment (Pipe-Smoking-Rabbit) [inline src/components/PostComment.jsx:23]: I'd recommend trying to stay away from `alert` as it's a bit of an old fashioned approach these days and might get blocked by a user's popup-blocking extension if they have one

File: src/components/PostComment.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          {isLoading ? <div className='loader'></div> : null}
+          <label htmlFor='comment-text'>Write your comment here:</label>
+          <textarea
+            id='comment-text'
+            onChange={onChangeHandler}
+            value={comment}
+            name='newComment'
+            rows='5'
+          />
+          <button disabled={!comment || isLoading}>Post comment</button>
```
Comment (Pipe-Smoking-Rabbit) [inline src/components/PostComment.jsx:49]: Really nice logic to disable the button 👏

**Review by Pipe-Smoking-Rabbit — COMMENTED** [review]
This both looks and feels great to use! 🏆 
Something that could improve user experience is if the loading spinner doesn't push the comment posting form down when it appears 💡 
I'd try and steer away from the `alert` personally and try to implement something similar to your error response - but telling the user it's been posted ✅

File: src/components/PostComment.jsx (hunk 1 above)
Comment (golovko) [reply src/components/PostComment.jsx:23]: yeah, using it just for debug - i can see preloader while it is stays opened =))

**Pull request merged**